app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -124,6 +127,25 @@ def handle_remember_device_preference(remember_device_preference)
     save_remember_device_preference(remember_device_preference)
   end
 
+  def mfa_selection_attempt_count(auth_method)
+    user_session[:mfa_attempts] ||= {}
+    auth_method = auth_method.gsub('personal_key', 'personal-key')
```

Comment:
Adding 'personal_key' to a session would trigger SessionEncryptor::SensitiveKeyError even though here we're just counting attempts with that as the key. 
Interested in a more idiomatic way of handling this edge case.

Reply:
Maybe we could build support for something similar to [`FakeAnalytics`' `pii_like_keypaths`](https://github.com/18F/identity-idp/blob/74b19c4b4ab14ca9eaf1eb70171b0e857018d85c/spec/support/fake_analytics.rb#L49) to bypass the default error handling.